### PR TITLE
Room

### PR DESCRIPTION
**Pull request by jalyxis55**
Merge updated room types

### CODE REVIEW

File: src/artifact_room.py [hunk 1]
```diff
@@ -5,15 +5,17 @@
...
-        """@ReturnType Artifact Room"""
-        self.___item = item
-        """@AttributeType Item"""
+        self.item = item
         super(ArtifactRoom, self).__init__(x, y)
 
-    def addArtifact(self):
-        """@ReturnType void"""
-        pass
+	def introText(self):
```
Comment (patrickv83) [inline src/artifact_room.py]: Wonky indent

File: src/artifact_room.py [hunk 2]
```diff
@@ -5,15 +5,17 @@
...
+		# print room description
+		return
 
     def modifyPlayer(self, player):
-        pass
+		# update player inventory with item found in room 
+        return player
+		
+	def getCoordinates(self):
```
Comment (patrickv83) [inline src/artifact_room.py]: And here

File: src/enemy_room.py [hunk 3]
```diff
@@ -1,10 +1,24 @@
...
+	# EnemyRoom constructor
     def __init__(self, x, y, enemy):
-        """@ReturnType EnemyRoom"""
-        self.___enemy = enemy
-        """@AttributeType Enemy"""
+        self.enemy = enemy
+		super(EnemyRoom, self).__init__(self, x, y)
+		
+	# introText prints room description
+	def introText(self):
```
Comment (patrickv83) [inline src/enemy_room.py]: And here

File: src/enemy_room.py [hunk 4]
```diff
@@ -1,10 +1,24 @@
...
+		return
+	
+	# modifyPlayer
+	def modifyPlayer(self, player):
```
Comment (patrickv83) [inline src/enemy_room.py]: here

File: src/enemy_room.py [hunk 5]
```diff
@@ -1,10 +1,24 @@
...
+		# Update player with results of enemy interaction
+		return player
+	
+	# Postcondition: returns room coordinates
+	def getCoordinates(self):
```
Comment (patrickv83) [inline src/enemy_room.py]: and here

**Review by patrickv83 — CHANGES_REQUESTED** [review]

**Review by jalyxis55 — COMMENTED** [review]
thanks, done

**Review by patrickv83 — APPROVED** [review]

**Pull request merged**